src/cli/scaffold/rust/lib.rs

```diff
@@ -0,0 +1,10 @@
+#[macro_use]
+extern crate hdk;
+extern crate serde;
+#[macro_use]
+extern crate serde_derive;
+#[macro_use]
+extern crate serde_json;
+extern crate boolinator;
+
+// see https://holochain.github.io/rust-api/0.0.1/hdk/ for info on using the hdk library
```

Comment:
Should this file stay this way? Or should we put in a basic zome and entry definition?

Reply:
I'm not sure... what do you think?

Reply:
I think we have to put in some basic zome definition

Reply:
I agree, that'd be good, and also I think be required to get tests passing